distributed/asyncio.py

```diff
@@ -72,43 +61,45 @@ class AioClient(Client):
 
     Examples
     --------
-    Provide cluster's scheduler address on initialization.
-
-    >>> client = AioClient('127.0.0.1:8786')  # doctest: +SKIP
-
-    The AioClient can receive an asyncio loop explicitly:
-
-    >>> loop = asyncio.get_event_loop()  # doctest: +SKIP
-    >>> client = AioClient('127.0.0.1:8786', loop=loop)  # doctest: +SKIP
+    Provide cluster's scheduler address on initialization::
 
-    Start the client:
+        client = AioClient('127.0.0.1:8786')
 
-    >>> await client.start()  # doctest: +SKIP
```

Comment:
Not valid syntax. I decided to show code blocks instead of an interactive session.